src/services/ApiService/MyCryptoApi/MyCryptoApi.ts

```diff
@@ -32,4 +32,14 @@ export default class MyCryptoApiService {
       return {};
     }
   };
+
+  public getSchemaMeta = async (): Promise<Record<TxType, ITxTypeMeta>> => {
+    try {
+      const { data } = await this.service.get<Record<TxType, ITxTypeMeta>>('schemas.json');
+      return data;
+    } catch (e) {
+      console.debug('[MyCryptoApiService]: Fetching schemas meta failed: ', e);
```

Comment:
Don't use `console.debug`

Reply:
We use this format on many other external API calls to help debug when there are issues in prod. Is there a reason to not use them?